toolkit/tools/internal/file/file.go

```diff
@@ -429,3 +430,14 @@ func EnumerateDirFiles(dirPath string) (filePaths []string, err error) {
 	}
 	return filePaths, nil
 }
+
+func CommandExists(name string) (bool, error) {
```

Comment:
Could you also add tests for the new function?

Reply:
I'm guessing all systems will have at least `ls` or other similar command we can test against.

Reply:
Fixed.